pyshepseg/tiling.py

```diff
@@ -311,7 +312,7 @@ def doTiledShepherdSegmentation(infile, outfile, tileSize=DFLT_TILESIZE,
         bandNumbers=None, subsamplePcnt=None, maxSpectralDiff='auto', 
         imgNullVal=None, fixedKMeansInit=False, fourConnected=True, 
         verbose=False, simpleTileRecode=False, outputDriver='KEA',
-        spectDistPcntile=50):
+        creationOptions=[], spectDistPcntile=50):
```

Comment:
The docstring for `Driver.Create` seems to indicate that `None` is the default option but actually that raises an error....